hack/ci/e2e.sh

```diff
@@ -130,8 +130,6 @@ run_tests() {
     echo "Patched CoreDNS config:"
     echo "${fixed_coredns}"
     kubectl apply -f - <<< "${fixed_coredns}"
-    # restart the pods
-    kubectl -n kube-system delete pods -l k8s-app=kube-dns
```

Comment:
just to confirm: we only ever needed this for unreleased versions of kubernetes, right? (IE, 1.16 development commits)

Reply:
exactly, this was a workaround for the 1.5.0 CoreDNS bump. AFAIK it was temporary because 1.16 released with 1.6.2